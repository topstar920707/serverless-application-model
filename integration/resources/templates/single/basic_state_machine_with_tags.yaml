Resources:
  MyStateMachine:
    Type: AWS::Serverless::StateMachine
    Properties:
      Definition:
        Comment: A Hello World example of the Amazon States Language using Pass states
        StartAt: Hello
        States:
          Hello:
            Type: Pass
            Result: Hello
            Next: World
          World:
            Type: Pass
            Result: World
            End: true
      Policies:
        - Version: "2012-10-17"
          Statement:
            - Effect: Deny
              Action: "*"
              Resource: "*"
      Tags:
        TagOne: ValueOne
        TagTwo: ValueTwo
      Tracing:
        Enabled: true

Outputs:
  MyStateMachineArn:
    Description: ARN of the state machine
    Value:
      Ref: MyStateMachine
