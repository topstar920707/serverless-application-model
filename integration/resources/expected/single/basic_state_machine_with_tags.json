[
    { "LogicalResourceId":"MyStateMachine", "ResourceType":"AWS::StepFunctions::StateMachine"},
    { "LogicalResourceId":"MyStateMachineRole", "ResourceType":"AWS::IAM::Role" }
]
